react/src/App.jsx

```diff
@@ -78,6 +78,9 @@ function App() {
     window.Helpshift("updateHelpshiftConfig");
```

Comment:
We should add comment at the first occurrence of this API to convey the usage of this to customer devs